datafusion/core/src/test/exec.rs

```diff
@@ -643,3 +637,139 @@ pub async fn assert_strong_count_converges_to_zero<T>(refs: Weak<T>) {
     .await
     .unwrap();
 }
+
+/// Execution plan that emits streams that panics.
+///
+/// This is useful to test panic handling of certain execution plans.
+#[derive(Debug)]
+pub struct PanicingExec {
```

Comment:
```suggestion
pub struct PanickingExec {
```

Reply:
I was trying to mirror the spelling of the rust `panic!` macro that doesn't have the `k` -- Maybe I will rename this just to `PanicExec` 🤔

Reply:
That is why I plan to rename it `PanicStream` to avoid such English peculiarities 😅

Reply:
in 3f80690f7